django_stubs_ext/setup.py

```diff
@@ -10,9 +10,11 @@
     "typing-extensions",
 ]
 
+# NB! Keep version in sync with django-stubs.
```

Comment:
```suggestion
# NB! Keep major.minor version in sync with django-stubs.
```

Reply:
Ok, I don't see a problem with that! Releasing empty releases is not a big problem. Getting out of sync is.

Reply:
> I think we should keep all major.minor.patch in sync.

Oh I remember now what my main reason for doing this was: if we keep them in sync, don't need separate `git tag`s for django-stubs and django-stubs-ext, and thus also don't need seperate relase notes.